app/assets/stylesheets/application.scss

```diff
@@ -58,16 +302,18 @@
     margin-right: auto;
   }
 
-  .top-bar .name h4 a:before {
-    background-image: url('MKX-Logo-Icon.jpg');
-    background-repeat: no-repeat;
-    content: "[]";
-    display: inline-block;
-    height: 57px;
-    position: relative;
-    width: 60px;
+  #register-button {
+    column-rule-color: rgba(0,0,0,0.30);
```

Comment:
Properties should be ordered background-color, column-rule-color, margin-right<br>Commas in function arguments should be followed by a single space